scio-jdbc/src/main/scala/com/spotify/scio/jdbc/package.scala

```diff
@@ -0,0 +1,46 @@
+package com.spotify.scio
+
+import org.apache.beam.sdk.io.jdbc.JdbcIO.{PreparedStatementSetter, RowMapper, StatementPreparator}
+
+package object jdbc {
+
+  val TEST_READ_TABLE_NAME = "table_read"
+  val TEST_WRITE_TABLE_NAME = "table_write"
+
+  /**
+    * Options require to create a connection with remote database.
+    * @param username database username
+    * @param password database password
+    * @param connectionUrl connection url i.e "jdbc:mysql://[host]:[port]/db?"
+    * @param driverClassName database driver class name
+    */
+  case class DbConnectionOptions(username: String,
+                                 password: String,
+                                 connectionUrl: String,
+                                 driverClassName: String)
+
+  /**
+    * Values need to initiate read connection to database and Convert it to given type.
+    * @param dbConnectionOptions Options need to create a database connection.
+    * @param query JDBC query string
+    * @param statementPreparator sql Statement Preparator
+    * @param rowMapper sql Row mapper to read from ResultSet
+    * @tparam T serializable type
+    */
+  case class JdbcReadOptions[T](dbConnectionOptions: DbConnectionOptions,
+                                query: String,
+                                statementPreparator: StatementPreparator,
```

Comment:
Wouldn't this be more scala friendly is this instead was something like `statementPreparator: (PreparedStatement) => Unit`?